### PR TITLE
Webpack-branch

### PR DESCRIPTION
**Pull request by ikoote1**
###TO DO LIST
- Added linter error files
- Created the working environment by installing webpack.
- Added html, CSS and Js files.
- Bundled the files with Webpack using the build command.
- Fixed errors in the code.
- Added and update the readme file.

Thanks for reviewing my work

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @ikoote1,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly refactor your readme file to show all the sections once instead of having to click "Details" dropdown just like the template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,176 @@
...
+
+### Git <a name="Git"></a>
+### Vs code<a name="Vs code"></a>
+### Javascript <a name="Javascript"></a>
+### HTML<a name="HTML"></a>
+### CSS <a name="CSS"></a>
+### webpack <a name="webpack"></a>
+
+<details>
+  #Client
```
Comment (julie-ify) [inline README.md]: - [x] I have to click "Details" for me to see the rest of the readme file. Kindly ensure all sections are displayed at once upon visit.

File: webpack.config.js [hunk 2]
```diff
@@ -1,26 +1,27 @@
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const path = require("path");
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
 module.exports = {
   module: {
     rules: [
```
Comment (julie-ify) [inline webpack.config.js]: - [x] **[OPTIONAL]** I would recommend you heed this warning and set a "mode" for the project.

  image

  ```
  module.exports = {
  mode: 'development',
  module: {
    ...
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ikoote1,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @ikoote1 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**